vue/src/modules/Workspace/Workspace.vue

```diff
@@ -55,13 +55,31 @@ export default {
     }
   },
   computed: {
-    ...mapState(['loading', 'workspaces'])
+    ...mapWorkspaceState({
+      loading: state => state.view.loading,
```

Comment:
@skavt Why are we using view page's loading in outer page ? It should have its respective loading attribute. `state.loading`